ceph_deploy/hosts/debian/install.py

```diff
@@ -30,11 +30,11 @@ def install(distro, version_kind, version, adjust_repos, **kw):
         distro.packager.add_repo_gpg_key(gpg.url(key, protocol=protocol))
 
         if version_kind == 'stable':
-            url = 'http://download.ceph.com/debian-{version}/'.format(
+            url = 'https://download.ceph.com/debian-{version}/'.format(
```

Comment:
@liewegas Just to follow up here.  I think we are looking for one change, which is to change:

`distro.packager.install('ca-certificates')`

into:

`distro.packager.install(['ca-certificates', 'apt-transport-https'])`